tests/behat/features/d7.field-collection.feature

```diff
@@ -4,8 +4,9 @@ Feature: Check that FieldCollectionTrait works for D7
   Background:
     Given I am logged in as a user with the "administrator" role
     And article content:
-      | title           | field_collection:field_fc_text_first;field_fc_text_second | field_collection_2:field_fc_text_third;field_fc_text_fourth |
-      | Test collection | test_text1;test_text2                                     | test_text3;test_text4                                       |
+      | title                       | field_collection:field_fc_text_first;field_fc_text_second | field_collection_2:field_fc_text_third;field_fc_text_fourth |
```

Comment:
currently, it is too hard to understand which field belongs to which field collection and which value belong to which field from the field collection.

Can you please update the field names in the fixture to be like
`field_fc1_text11`,`field_fc1_text12`, `field_fc2_text21`, `field_fc2_text22` 

and then update the values to be like 
```
fc1_text11_value111
fc1_text11_value112
fc1_text11_value113

fc1_text12_value121
fc1_text12_value122

fc2_text21_value211
fc2_text21_value212
fc2_text21_value213

fc2_text22_value221
fc2_text22_value222

```